client/camera/CameraMain.css

```diff
@@ -28,28 +28,88 @@
   flex: 1 0 auto;
 }
 
+:local(.note) {
+  font-size: 0.9em;
+}
+
 :local(.sidebar) {
   flex: 1 1 cameraMainSidebarWidth;
   color: white;
   font: 18px sans-serif;
   font-weight: 300;
 }
 
+:local(.sidebar) h1, h2, h3, h4, h5, h6 {
```

Comment:
Ah, this doesn't do what you think it does: you'd want

```css
:local(.sidebar) h1,
:local(.sidebar) h2,
:local(.sidebar) h3,
:local(.sidebar) h4,
:local(.sidebar) h5,
:local(.sidebar) h6
```

I'll fix this in a separate commit.

Reply:
ah, thanks! not used to doing css like this!